docs(DateInputs) add clear button docs (#2031)
telerik/blazor-docs: components/dateinput/overview.md

| `Max` | `DateTime` <br /> (`new DateTime(2099, 12, 31)`)| The latest allowed date that the user can type. |
| `Min` | `DateTime` <br /> (`DateTime(1900, 1, 1)`)| The earliest allowed date that the user can type. |
| `Placeholder` | `string` | The [`placeholder` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#placeholder) of the `input`. The placeholder will appear only if the component is bound to nullable `DateTime?` object, the `Value` is `null` and the component is not focused. Once the user focuses it to start typing, the `FormatPlaceholder` (default or [custom one](#format-placeholder)) will override the `Placeholder` to indicate the expected date format. |

> | `ShowClearButton` | `bool` | Defines if the user will have the option to clear the component value with an "**x**" button inside the input. |

Reviewer: * This is a more concise version of the sentence that you could apply to the rest of the files too.
* Bold (**x**) is used to indicate that this is an element rendered in the UI. Don't use quotes in addition to the bold font.
```suggestion
| `ShowClearButton` | `bool` | Defines if the user can clear the component value through an **x** button rendered inside the input. |
```
